.github/workflows/generator_generic_slsa3.yml

```diff
@@ -108,7 +108,7 @@ jobs:
       actions: read # Needed to read workflow info.
     steps:
       - name: Generate builder
-        uses: slsa-framework/slsa-github-generator/.github/actions/generate-builder@923a5a249d2ea820614a80486c9ef0857ced5321
+        uses: slsa-framework/slsa-github-generator/.github/actions/generate-builder@main
```

Comment:
Not sure if it's worth it but I think we could technically unpin `generate-builder` if we bring the `secure-builder-checkout` out into the reusable worflows.

In that case the rule would be that all actions are unpinned except `detect-env` and `secure-builder-checkout`.

Reply:
we could. But then we pass the same arguments to both `secure-builder-checkout` and `generate-builder` which seems a little weird. wdut?